core/src/test/scala/com/evolutiongaming/kafka/flow/ConsumerFlowSpec.scala

```diff
@@ -0,0 +1,178 @@
+package com.evolutiongaming.kafka.flow
+
+import cats.data.{NonEmptyMap, NonEmptySet}
+import cats.effect.{Resource, SyncIO}
+import cats.effect.concurrent.Ref
+import cats.syntax.all._
+import com.evolutiongaming.catshelper.LogOf
+import com.evolutiongaming.kafka.flow.kafka.Consumer
+import com.evolutiongaming.kafka.journal.ConsRecords
+import com.evolutiongaming.skafka.consumer.{ConsumerRecords, RebalanceListener}
+import com.evolutiongaming.skafka.{Offset, OffsetAndMetadata, Partition, Topic, TopicPartition}
+import munit.FunSuite
+
+import scala.concurrent.duration._
+
+class ConsumerFlowSpec extends FunSuite {
```

Comment:
CI is complaining about imports in this file:
https://github.com/evolution-gaming/kafka-flow/pull/130/checks?check_run_id=1590071724

Reply:
Fixed